src/app/app.component.scss

```
@@ -0,0 +1,7 @@
+.title {
+  color: #5e3030;
+  font-size: 20px;
+}
+.isredbox {
+  color: red;
```

use color variables from _variables.scss
use kebab-case for css styles
```suggestion
.red-box {
  color: red;
```